### ISSUE TITLE
Training BatchNorm and Only BatchNorm: On the Expressive Power of Random Features in CNNs

### ISSUE THREAD

**Issue by KeisukeShimokawa** [KeisukeShimokawa/papers-challenge#102]
### 論文へのリンク

[[arXiv:2003.00152] Training BatchNorm and Only BatchNorm: On the Expressive Power of Random Features in CNNs]

### 著者・所属機関

Jonathan Frankle, David J. Schwab, Ari S. Morcos

- MIT CSAIL
- CUNY Graduate Center, Initiative for the Theoretical Sciences
- Facebook AI Research

### 投稿日時（YYYY-MM-DD）

2020-02-29

**Comment by KeisukeShimokawa** [comment]
[reddit](https://www.reddit.com/r/MachineLearning/comments/fe06er/r_training_batchnorm_and_only_batchnorm_on_the/)

**Comment by KeisukeShimokawa** [comment]
[batchnorm implementation of official pytorch](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/batchnorm.py)
